server/src/main/java/org/elasticsearch/search/aggregations/metrics/GlobalOrdCardinalityAggregator.java

```diff
@@ -105,16 +105,23 @@ public ScoreMode scoreMode() {
     private class CompetitiveIterator extends DocIdSetIterator {
 
         private final BitArray visitedOrds;
+        private final SortedSetDocValues values;
```

Comment:
Use a different name here to avoid confusion?

Reply:
I remove this...it is not needed (see following commit)